test/testCSVConverter.js

```diff
@@ -410,6 +415,49 @@ describe("CSV Converter", function () {
     });
   });
 
+  it("should emit eol event when line ending is detected as CRLF", function (done) {
+    var testData = __dirname + "/data/dataNoTrimCRLF";
+    var rs = fs.createReadStream(testData);
+
+    var st = rs.pipe(new Converter());
+    var eolCallback = sandbox.spy(function (eol) {
+      assert.equal(eol, "\r\n");
+    });
+    st.on("eol", eolCallback);
+    st.on("end_parsed", function () {
+      assert.equal(eolCallback.callCount, 1, 'should emit eol event once');
+      done();
+    })
+  });
+
+  it("should emit eol event when line ending is detected as LF", function (done) {
+    var testData = __dirname + "/data/columnArray";
+    var rs = fs.createReadStream(testData);
+
+    var st = rs.pipe(new Converter());
+    var eolCallback = sandbox.spy(function (eol) {
+      assert.equal(eol, "\n");
+    });
+    st.on("eol", eolCallback);
+    st.on("end_parsed", function () {
+      assert.equal(eolCallback.callCount, 1, 'should emit eol event once');
+      done();
+    })
+  });
+
+  it("should not emit eol event when eol is specified", function (done) {
```

Comment:
Not sure what the best behavior is here. Does it make sense to always emit it?

Reply:
I think we need emit it if user subscribes to the event.